packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const BreakoutRoomIcon = createSvgIcon({
```

Comment:
I am not sure about naming there. FA has `door-open` for similar icon, https://fontawesome.com/icons/door-open?style=solid. Any ideas?

Reply:
Name is fine